docs/visualization/visualize_2p_responses.ipynb

```diff
@@ -38,7 +38,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.        # retrieves the stimulus table associated with current 'stim_name'
</pre><p>probably not necessary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/233/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>